apps/concierge_site/mix.exs

```diff
@@ -41,7 +41,8 @@ defmodule ConciergeSite.Mixfile do
       {:bamboo, "~> 2.2.0"},
       {:bamboo_phoenix, "~> 1.0.0"},
       {:bamboo_ses, "~> 0.3.0"},
-      {:ex_aws_sns, "~> 2.2.0"},
+      # TOOD: use official release when tagged
```

Comment:
suggestion (non-blocking): It might be worth calling out explicitly that the `check_if_phone_number_is_opted_out` functionality is what we are specifically wanting in the release (so that future us knows whether or not an acceptable release was indeed tagged).